server/config.go

```diff
@@ -1,12 +1,14 @@
 package server
 
 import (
+	"io/fs"
 	"time"
 )
 
 // Defines default config settings (excluding limits, see below)
 const (
 	DefaultListenHTTP                           = ":80"
+	DefaultListenUnixMode                       = 0777
```

Comment:
Why would 777 be the default? Can't you use the 700 or just the default umask?

Reply:
So that everything can connect to it by default, but I can change it.
As for the umask, I think it's already using it, which is why it defaults to 0755 without the `Chmod` call.
But for this particular case, the umask should be ignored, in favor of an explicit mode being set.
I could just set the service's umask to 0000 and "fix" my problem, but then every other file created by it would be world-writable.